sdk/typescript/src/types/objects.ts

```diff
@@ -43,13 +64,25 @@ export const RawObjectRefSchema = tuple([
   ObjectDigestSchema,
 ]);
 
+export const ObjectContentSchema = any();
```

Comment:
This `any` for example could be something like:
```
type Contents = {
    'fields': Record<string, Contents | string | boolean | number ....(all possible value types)>,
    'type': string,
}
```

Not sure if that works with `superstruct` though